turbo/src/main/kotlin/dev/hotwire/turbo/config/TurboPathConfigurationLoader.kt

```diff
@@ -30,29 +28,31 @@ internal class TurboPathConfigurationLoader(val context: Context) : CoroutineSco
         loadCachedConfigurationForUrl(url, onCompletion)
 
         launch {
-            repository.getRemoteConfiguration(url)?.let {
-                onCompletion(load(it))
-                cacheConfigurationForUrl(url, load(it))
+            repository.getRemoteConfiguration(url)?.let { remoteConfigJson ->
+                repository.parseFromJson(remoteConfigJson)?.let { config ->
```

Comment:
If this is the same repository in both cases, should the parsing be simply a part of the outer method?

Reply:
good point, thanks 👍